### ISSUE TITLE
tflearn examples segmentation fault

### ISSUE THREAD

**Issue by remkonoteboom** [tflearn/tflearn#703]
I tried out the titanic example, but running gives a segmentation fault.  I reduced the number of lines and it will segfault with the following:

import tflearn
net = tflearn.input_data(shape=[None, 6])
model = tflearn.DNN(net)

It does come back and I can print the "model" variable so it seems to occur on exit of this simple script.

On the "titanic" example, it will segfault on:
model.fit(data, labels, n_epoch=10, batch_size=16, show_metric=True)

I traced this to:

 self.summ_writer = writer_summary(
     self.tensorboard_dir + run_id, self.session.graph)

in helpers/trainers.py.  "wrtier_summary" prints as follows:

<class 'tensorflow.python.summary.writer.writer.FileWriter'>

I have also tried finetuning.py and it segfaults as well.

I am running:
OS: Fedora 19 (old, but everything else works)
Python v2.7.5
tensorflow: v1.0.1
numpy: 1.12.1
scipy: 0.19.0

I am running this in a virtualenv environment and everything was installed with pip in this environment.  Is this a known issue with this environment?

**Comment by aymericdamien** [comment]
Segmentation fault are harder to check :/ 
Have you verify that you have permissions to write to `tensorboard_dir` folder?

**Comment by remkonoteboom** [comment]
If you mean the folder /tmp/tflearn_logs, which I believe is the default, then yes:

(tensorflow)[tactic@localhost scripts]$ find /tmp/tflearn_logs/
/tmp/tflearn_logs/
/tmp/tflearn_logs/2V1UCL
/tmp/tflearn_logs/PB3LSU

Afer running the script, it does create folders here ... but they are empty.  So no logs; only a new folder based on the run id.

**Comment by remkonoteboom** [comment]
FYI: I finally got around to trying this out with Fedora 25 and it worked.
